vitest.setup.ts

```diff
@@ -34,6 +37,20 @@ vi.stubGlobal('useRuntimeConfig', () => ({
   },
 }))
 
+export const mockI18n = vi.fn((options?: NuxtI18nOptions) => ({
+  locale: options?.defaultLocale || 'en',
+  locales: options?.locales || supportedLocales,
+  ...options,
+}))
+vi.stubGlobal('useI18n', mockI18n)
+
+const i18n = createI18n({
+  locale: 'en',
+  legacy: false,
+  messages: localeMessages,
+})
+
+config.global.plugins = [i18n]
```

Comment:
Why do we need to call `createI18n` here? Can't we just call `mockI18n`? Sorry but I'm confused here